### ISSUE TITLE
What widely available board can I get for this?

### ISSUE THREAD

**Issue by Zerith** [ufrisk/pcileech#23]
What is the cheapest and/or most widely available board I can buy to make a POC for this?
I looked at USB3380-EVB from bplus but they are very unresponsive and the handling times and shipping will be too long for me..

**Comment by ufrisk** [comment]
I'm sorry, but there isn't a supplier that I'm aware of that have any boards available now that BPlus sold out (at least at this moment). I'd say your best bet is unfortunately to wait for BPlus to restock (if they do that) or purchase a used board from someone.

PCILeech support might expand to more hardware in the future, I have some PoC code for [@d_olex](https://twitter.com/d_olex) yet unreleased FPGA project already in PCILeech, but it's slow at the moment; and his code isn't released publicly yet.

Also, there are some other people looking into creating boards that will support PCILeech, but it's hard for me to know when/if such projects would materialize into a finished product.

I'm sorry, I know it's not the answer you wished for, but it's unfortunately how things are at the moment.

**Comment by Zerith** [comment]
Do you think the following can work:

If I get a PCIE Firewire card like [this one ](https://www.amazon.com/Rosewill-FireWire-1394a-Components-RC-504/dp/B004F3DM6C)  and connect my laptop through firewire, do you think It is possible to somehow patch the code to make it send the DMA commands from my laptop through the firewire?

**Comment by ufrisk** [comment]
Firewire DMA is blocked by quite a few OS'es nowdays though; but please have a look at [inception](https://github.com/carmaa/inception) if you haven't already done it.

I've looked into possibly supporting Firewire on PCILeech as well, but if I understand it correctly it would require some kind of kernel driver on the attacking computer - which makes it overly complicated for me to add support.

**Comment by ufrisk** [comment]
Update: Please see my additions about the SP605 FPGA dev board and together with the FT601 USB3 addon board.

It's expensive, it's bulky, but it's more capable than the USB3380 and it's available.
